### ISSUE TITLE
100 Automations: Update alt text for 100 Automations image to adhere to WCAG

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#3868]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text for the `100 Automations` image so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the alt value within `_projects/100-automations.md`:

**From:**
```
alt: '100 Automations logo depicted as a 1 and two intermeshed gears surrounded by a open left bracket, forward slash and closing right bracket to indicate the 100 automations are in code, followed by the word Automations.'
```

**To:**
```
alt: '100 Automations'
```

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
-  100 Automations Page: https://www.hackforla.org/projects/100-automations

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE:  `_projects/100-automations.md`
- See the source file code on the repo: https://github.com/hackforla/website/blob/gh-pages/_projects/100-automations.md
- This issue is part of epic #3866

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by adevlinb** [comment]
ETA: Tuesday by 6PMPT
Availability: Tuesday 6PMPT - 7PMPT
